lib/auth0/mixins/httpproxy.rb

```diff
@@ -20,6 +20,8 @@ module HTTPProxy
                      call(:get, url(safe_path), timeout, get_headers)
                    elsif method == :delete
                      call(:delete, url(safe_path), timeout, add_headers({params: body}))
```

Comment:
what is this `params:body` header for? I guess you meant to send a query instead?

Reply:
Oddly, sending it as `params:` converts it to a query ... one of the many quirks of this HTTP library